src/config/errors.sol

```diff
@@ -120,22 +123,36 @@ error OT_InvalidExpiry();
 /// @dev put and call should not contain "short stirkes"
 error OT_BadStrikes();
 
-// Chainlink Pricer Errors
+// Chainlink oracle
 
 error CL_AggregatorNotSet();
```

Comment:
Does it make sense to move these errors into the oracle contract? What if someone is using a different price feed, i dont think we want this errors file to be the dumping grounds.

Reply:
yeah agree, start to see this happen while multiple things are being added in parallel.  will update